test/fw/ptl/lib/ptl_wrappers.py

```diff
@@ -2764,18 +2764,37 @@ def manager(self, cmd, obj_type, attrib=None, id=None, extend=None,
         if id is None and obj_type == SERVER:
             id = self.pbs_conf['PBS_SERVER']
         bs_list = []
-        if cmd == MGR_CMD_DELETE and oid is not None and rc == 0:
-            for i in oid:
-                if obj_type == MGR_OBJ_HOOK and i in self.hooks:
-                    del self.hooks[i]
-                if obj_type in (NODE, VNODE) and i in self.nodes:
-                    del self.nodes[i]
-                if obj_type == MGR_OBJ_QUEUE and i in self.queues:
-                    del self.queues[i]
-                if obj_type == MGR_OBJ_RSC and i in self.resources:
-                    del self.resources[i]
-                if obj_type == SCHED and i in self.schedulers:
-                    del self.schedulers[i]
+        if cmd == MGR_CMD_DELETE and oid is not None:
+            if rc == 0:
+                for i in oid:
+                    if obj_type == MGR_OBJ_HOOK and i in self.hooks:
+                        del self.hooks[i]
+                    if obj_type in (NODE, VNODE) and i in self.nodes:
+                        del self.nodes[i]
+                    if obj_type == MGR_OBJ_QUEUE and i in self.queues:
+                        del self.queues[i]
+                    if obj_type == MGR_OBJ_RSC and i in self.resources:
+                        del self.resources[i]
+                    if obj_type == SCHED and i in self.schedulers:
+                        del self.schedulers[i]
+            else:
+                if obj_type == MGR_OBJ_RSC:
+                    res_ret = self.du.run_cmd(cmd=[
+                        os.path.join(
+                            self.pbs_conf['PBS_EXEC'],
+                            'bin',
+                            'qmgr'),
+                        '-c',
+                        "list resource"],
```

Comment:
Can we use self.server.mananger(MGR_CMD_LIST, RSC) here?

Reply:
No that will through old list

Reply:
Won't it go out to PBS and refresh the list?  Maybe I'm confused on what MGR_CMD_LIST does.

Reply:
No, It won't , It just returns a self.resources list